### DISCUSSION TITLE
Anyone can login with GitHub auth

### DISCUSSION THREAD

**Discussion by toby-king** [sveltia/sveltia-cms#252] [Q&A] [answered]
Recently migrated to Sveltia from Decap and installed the Github backend on Netlify as instructed in the guide. However, it now means that anyone with a GitHub account can login to the admin panel - is there a way to restrict access?

**Comment by kyoshino** [comment]
Have you tried Decap CMS with the GitHub backend? Sveltia CMS uses the same OAuth process, so I don’t think this topic is specific to Sveltia. If your repository is public, there should be no concern. People can see your content with or without the CMS. Only people who have been invited to the repo and granted write access can commit. If your repository is private, only collaborators should be able to access.

**Reply by kyoshino** [reply]
Looks like Netlify/Decap CMS checks the user’s write permission at some point. Even though non-collaborators being able to sign in is not a security risk for public repositories, it is awkward UX-wise. So I’ll implement the same check.

**Comment by kyoshino** [comment, marked as answer]
Fixed in [v0.49.2](https://github.com/sveltia/sveltia-cms/releases/tag/v0.49.2).
